clients/cli/tests/helpers.rs

```diff
@@ -75,36 +62,237 @@ pub async fn setup() -> Env {
     };
     solana_config.save(&config_file_path).unwrap();
 
-    Env {
-        rpc_client: Arc::new(validator.get_async_rpc_client()),
+    TestEnv {
+        payer,
+        rpc_client,
         config_file_path,
-        unwrapped_mint: unwrapped_mint.0,
-        unwrapped_token_program: UNWRAPPED_TOKEN_PROGRAM,
-        wrapped_token_program: WRAPPED_TOKEN_PROGRAM,
-        _validator: validator,
         _keypair_file: keypair_file,
         _config_file: config_file,
+        _validator: validator,
     }
 }
 
-pub fn setup_mint(mint_authority: &Pubkey) -> (Pubkey, AccountSharedData) {
-    let state = spl_token::state::Mint {
-        decimals: 8,
-        is_initialized: true,
-        supply: 1_000_000_000,
-        mint_authority: COption::Some(*mint_authority),
-        freeze_authority: COption::None,
-    };
-    let mut data = vec![0u8; spl_token::state::Mint::LEN];
-    state.pack_into_slice(&mut data);
+pub async fn create_unwrapped_mint(
+    rpc_client: &Arc<RpcClient>,
+    payer: &Keypair,
+    token_program_addr: &Pubkey,
+) -> Pubkey {
+    let mint_account = Keypair::new();
+    let rent = rpc_client
+        .get_minimum_balance_for_rent_exemption(SplTokenMint::LEN)
+        .await
+        .unwrap();
 
-    let lamports = Rent::default().minimum_balance(data.len());
+    let blockhash = rpc_client.get_latest_blockhash().await.unwrap();
 
-    let account = Account {
-        lamports,
-        data,
-        owner: UNWRAPPED_TOKEN_PROGRAM,
-        ..Default::default()
-    };
-    (Pubkey::new_unique(), account.into())
```

Comment:
Taking action on some feedback in last PR: https://github.com/solana-program/token-wrap/pull/33#pullrequestreview-2707626189